internal/service/advancedcluster/resource_advanced_cluster_migration_test.go

```diff
@@ -47,7 +47,7 @@ func TestMigAdvancedCluster_replicaSetAWSProviderUpdate(t *testing.T) {
 	)
 
 	resource.ParallelTest(t, resource.TestCase{
-		PreCheck:     mig.PreCheckBasicSleep(t),
+		PreCheck:     acc.SkipIfTPFAdvancedCluster(t, mig.PreCheckBasicSleep(t)),
```

Comment:
what about calling the func in the beginning of the tests instead of inside the test case? e.g. like in: https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/internal/service/encryptionatrest/resource_migration_test.go#L186

Reply:
agree. Done in 053deb331cb7c3189a7de3ffa47e597ba5721ac1